lightningd/log.c

```diff
@@ -558,70 +558,60 @@ static void maybe_notify_log(struct logger *log,
 		notify_log(log->log_book->ld, l);
 }
 
+static void add_one_log_entry(struct logger *log, enum log_level level,
+			      const struct node_id *node_id, bool call_notifier,
+			      char *log_msg)
```

Comment:
`const char *` - we always use const when we can (it's a C bug that it's not the default!)

Reply:
Done